src/libs/actions/Policy/Policy.ts

```diff
@@ -2164,6 +2165,13 @@ function createWorkspace(
         shouldAddOnboardingTasks,
         companySize,
     );
+
+    // For test drive receivers, we want to complete the createWorkspace task in concierge, instead of #admin room
+    if (introSelected?.choice === CONST.ONBOARDING_CHOICES.TEST_DRIVE_RECEIVER) {
+        const createWorkspaceTaskReport = introSelected.createWorkspace ? {reportID: introSelected.createWorkspace} : undefined;
+        completeTask(createWorkspaceTaskReport);
+    }
```

Comment:
Thanks! This does work, but seeing it now I realize that we should not really be calling two API requests here. 

Ideally, we complete these tasks in `CreateWorkspace`. Let me see what that would take before we do this change? 

I think having this checked is not so urgent that it would be good to look into a BE solution.

Reply:
I'm guessing we would still have this logic but then pass:

```js
const parameters: CompleteTaskParams = {
    taskReportID,
    completedTaskReportActionID: completedTaskReportAction.reportActionID,
};
 ``` 
 to `CreateWorkspace` instead and have it perform the complete task part. Or something like that...

Reply:
Yeah! That's what I'm thinking! I'll look into the BE changes today.

Reply:
Oh actually it could be that this isn't necessary. If we end up with only creating the task once the backend might be able to already locate this task... Let me confirm.

Reply:
And we should include the optimistic data from completeTask now in the CreateWorkspace data.